public/editor/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -290,7 +290,7 @@ define(function(require) {
 
     // Programmatic change to tutorial vs. regular preview mode from Bramble
     bramble.on("tutorialVisibilityChange", function(data) {
-      if(data.visibility) {
+      if(data.visible) {
```

Comment:
Interesting, nice catch.  Looks like I documented this incorrect in https://github.com/mozilla/brackets/commit/b31163e3a7 (the README needs a fix in Brackets, can you do that too?)